src/scvmm/HISTORY.rst

```diff
@@ -2,6 +2,8 @@
 
 Release History
 ===============
+1.2.0
+* Add command `az scvmm create-from-machines`.
```

Comment:
Why not consider designing the command as `az scvmm create --source Machines`? 
Perhaps in the future, it can support more different sources without creating new commands, which may have better scalability